src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryTokenStore.cs

```diff
@@ -69,9 +69,13 @@ public void SaveCookieToken(HttpContext httpContext, string token)
             Debug.Assert(httpContext != null);
             Debug.Assert(token != null);
 
-            var options = new CookieOptions();
-            options.HttpOnly = true;
-            options.Domain = _options.CookieDomain;
+            var options = new CookieOptions
+            {
+                HttpOnly = true,
+                Domain = _options.CookieDomain,
+                SameSite = SameSiteEnforcementMode.Strict
```

Comment:
Spoke offline, I think this should remain as strict as we expect the cookie to be created by requests from the same site as subsequent requests. There's no need to make this configurable.